include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Hmm.  Actually there might be a way to bypass the linker errors in the mixed case.  I'll have to think about that tomorrow though, it's getting pretty late here.